deeplinkdispatch-processor/build.gradle

```diff
@@ -17,7 +17,6 @@ sourceSets {
 
 dependencies {
   implementation project(':deeplinkdispatch-base')
-  implementation 'com.google.auto.service:auto-service:1.0-rc3'
```

Comment:
Given that the right META-INF files are already added here, there's not much reason to keep the dependency.  In my experience (author of https://github.com/keepsafe/dexcount-gradle-plugin) these files are among the most stable part of the codebase - I don't think I've ever needed to change them.  IMO, AutoService doesn't really pay for itself unless you're also making use of `BasicAnnotationProcessor`, etc.

If you decide you _do_ want to keep this, I'd suggest switching this dependency to `implementation 'com.google.auto.service:auto-service-annotations:1.0-rc6'`, and adding another on `annotationProcessor 'com.google.auto.service:auto-service:1.0-rc6'`, to keep the runtime classpath cleaner.  (I've noticed that AutoService seems to spill over from this project into ours because of the current configuration.  I'm confused by this, but when I modified the deeplink processor locally this way, AutoService was no longer being run in our project.)

Reply:
Those are good points - I would also be fine with using explicit `META-INF` files like you had. I don't feel too strongly.

Reply:
I have changed it back to using META-INF when I rebased with master. Seems a lot less likely to break going forward, and is easy to manage in a project with a single annotation processor.

IntelliJ also is quite smart about formatting the file, and if the file is wrong, the compiler unit tests will fail